HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -103,6 +104,9 @@ final class InvoicesListViewModel {
         self.paymentComponentsController = paymentComponentsController
     }
     
+    func viewDidLoad() {
```

Comment:
[clarfification]: Do we still need this function?

Reply:
Yes. Actually in the next PR, there we will load the payment providers.

Reply:
Thanks for your answer.